mount/losetup_linux.go

```diff
@@ -32,8 +35,26 @@ const (
 	loopDevFormat   = "/dev/loop%d"
 
 	ebusyString = "device or resource busy"
+
+	LOOP_CONFIGURE = 0x4c0a //nolint: revive
 )
 
+type LoopConfig struct {
+	Fd        uint32
+	BlockSize uint32
+	Info      unix.LoopInfo64
+	Reserved  [8]uint64
+}
+
+func ioctlConfigure(fd int, value *LoopConfig) error {
+	_, _, err := syscall.Syscall(syscall.SYS_IOCTL, uintptr(fd), uintptr(LOOP_CONFIGURE), uintptr(unsafe.Pointer(value)))
```

Comment:
Is this something that could eventually be upstreamed to golang.org/x/sys?

Reply:
Yeah, It might be useful to add it in there. I'll check what is the submission flow for it.

Reply:
I know maintainers there have been quite receptive to such changes, but much of the code is generated, so it may indeed require a small amount of looking at existing examples to see how to create the patch. @tklauser (👋 sorry for the random ping ❤️) may be able to provide some guidance if needed